pkg/storage/tsdb/config.go

```diff
@@ -152,12 +152,12 @@ type BucketStoreConfig struct {
 
 // RegisterFlags registers the BucketStore flags
 func (cfg *BucketStoreConfig) RegisterFlags(f *flag.FlagSet) {
-	f.StringVar(&cfg.SyncDir, "experimental.tsdb.bucket-store.sync-dir", "tsdb-sync", "Directory to place synced tsdb indicies.")
+	f.StringVar(&cfg.SyncDir, "experimental.tsdb.bucket-store.sync-dir", "tsdb-sync", "Directory to store synched TSDB index headers.")
```

Comment:
Can we just use `synchronized` or `fetched`? I don't remember seeing `synched` before, but apparently it's a common variant.

Reply:
Ok. Picked `synchronized` (cause it's more then just fetching, given it also offload removed blocks).